kvm/kvm.go

```diff
@@ -19,4 +20,5 @@ type KVM struct {
 	Masters         []node.Node                     `json:"masters" yaml:"masters"`
 	Network         network.Network                 `json:"network" yaml:"network"`
 	Workers         []node.Node                     `json:"workers" yaml:"workers"`
+	NTP             ntp.NTP                         `json:"ntp" yaml:"ntp"`
```

Comment:
This should be alphabetical? Unless the capital T changes that. Flagging it anyway :D